src/Components/Scenes/RequestAssets/RequestedBy/RequestedBy-Container.js

```diff
@@ -0,0 +1,11 @@
+import { connect } from 'react-redux';
+import RequestedBy from './RequestedBy';
+
+// This will get updated when Auth is done
```

Comment:
Maybe tag it with a `TODO` just in case 👍

Reply:
added  `TODO`